lib/experimental/Widgets/Content/TasksList/TaskItem/index.tsx

```diff
@@ -0,0 +1,63 @@
+import { Badge } from "@/components/Information/Badge"
+import { Counter } from "@/experimental/Information/Counter"
+import { DottedCircle, InProgressTask } from "@/icons"
+import { useCallback } from "react"
+
+export type TaskStatus = "in-progress" | "due" | "no-due"
+export interface Task {
+  id: number | string
+  text: string
+  badge?: { text: string; isPastDue?: boolean }
+  counter?: number
+}
+
+export type TaskItemProps = {
+  task: Task
+  status: TaskStatus
+  onClick?: (task: Task) => void
+  hideIcon?: boolean
+}
+
+export function TaskItem({
+  task,
+  status,
+  onClick,
+  hideIcon = false,
+}: TaskItemProps) {
+  const handleClick = useCallback(() => {
+    onClick?.(task)
+  }, [onClick, task])
+
+  return (
+    <div
+      className={`flex flex-row items-center gap-2 border border-solid border-transparent p-2 ${
+        onClick
+          ? "cursor-pointer hover:bg-f1-background-tertiary focus:border-f1-background-selected-bold focus:outline-none"
+          : ""
```

Comment:
there is a `cn` helper to merge styles. It works like this (verify before applying)

```suggestion
      className={cn(
      "flex flex-row items-center gap-2 border border-solid border-transparent p-2", 
      onClick ? "cursor-pointer hover:bg-f1-background-tertiary focus:border-f1-background-selected-bold focus:outline-none" : undefined
      )}
```